### ISSUE TITLE
loss_value is NaN while training

### ISSUE THREAD

**Issue by cyberfag** [akaraspt/deepsleepnet#43]
Hi. During training the error occurs in _run_epoch of module trainer.py:  "Model diverged with loss = NaN". What could it be due to?

**Comment by akaraspt** [comment]
@cyberfag There can be due to several issues. Probably start from the data. Have you verified whether the input signals and labels are loaded properly?

**Comment by cyberfag** [comment]
@akaraspt Sorry, should've posted several days ago. The issue was solved by switching Python and Cuda/cuDNN versions to 3.6.0, 10.0, and 7, respectively. Now I encountered new problems but it's not related so the thread should be closed.
